docs/examples/Makefile.am

```diff
@@ -39,5 +39,9 @@ exampleprofile_DATA     = \
 exampleudevrulesdir = $(datarootdir)/@PACKAGE@/examples/config/rules.d
 exampleudevrules_DATA = \
 			rules.d/90-ipc-persistent-th.rules
+	
+examplelogdir	= $(datarootdir)/@PACKAGE@/examples/config/ftylog.d
+examplelog_DATA = \
+                   ftylog.d/10-ftylog.cfg
```

Comment:
Does the logger support loading several configurations (the `dirname.d/sortablename.conf` paradigm is for that)? 

If not, you don't need `.d` in the dirname and the number in the filename, which keeps the code simpler and similar to the names in the final deliverable.

Reply:
@jimklimov, if several agents provide their own log config file, yes it make sense to have this dirname.d folder available.
At the end all  log config files would be pushed in /etc/fty/ftylog.cfg

Reply:
@geraldguillaume : just in case - do you mean here the `dirname.d` concept at run-time? Can the different agents load and merge settings from several configuration files (e.g. the common config and their own customization if defined - and perhaps distributed with those agents as part of their codebase)?